### DISCUSSION TITLE
I can't deploy a docker compose (a stack) on Portainer

### DISCUSSION THREAD

**Discussion by Aissasa** [qdm12/gluetun#2508] [Q&A]
Hello, a noob here that would love some help please. 
So as the title says, I can't for the life of me figure out what im missing in my config, I followed what this guy is doing [here](https://github.com/GitHubMilo/vpn_downloads/blob/main/docker-compose.yml), and adapted it to my environment.

So for context, I'm running a debian VM on proxmox, this VM has docker installed, and Portainer. The VM is routed through basic bridge and is accessible to my local network.

I'm trying to setup a servarr stack on this VM that accesses an SMB share (that i have setup on another VM), and I tried to route my torrents traffic through gluetun. I have a mullvad subscription and im trying to use those credentials.

So here is my current docker compose; this is a simplified version since I started banging my head on the wall trying different things:
https://pastebin.com/msxGSyS3

I do have an environment file for env variables, but here are the highlights:
`PUID=1000
PGID=1000
TZ=Europe/Stockholm
ROOT=/svr/docker/servarr
ROOT_CONFIGS=/svr/docker/servarr/configs
SAMBA_SHARE=/mnt/smbshare
MULLVAD_COUNTRIES=Denmark,Sweden,Germany,Norway,Netherlands
QBT_WEBUI_PORT=8180`

so what happens is when i try to deploy this stack is, i get a consistent error that looks like follows:
`Failed to deploy a stack: Network media-stack_default Creating Network media-stack_default Created Container gluetun Creating Container gluetun Created Container qbittorrent Creating Container sonarr Creating Container radarr Creating Container sonarr Created Container radarr Created Container qbittorrent Created Container gluetun Starting Container gluetun Started Container qbittorrent Starting Container radarr Starting Container sonarr Starting Error response from daemon: failed to create task for container: failed to create shim task: OCI runtime create failed: runc create failed: unable to create new parent process: namespace path: lstat /proc/74118/ns/net: no such file or directory: unknown `

My noob brain is telling me that the same container is being triggered for creation multiple times for whatever reason (looking at the log, creating X container is called multiple times), but tbh, i'm out of ideas, hence why I'm here.
Worth noting that deploying the gluetun container on its own goes through, and i tried deploying the other containers but with WG container, also works fine, but once i try to combine the servarr containers and gluetun, shit hits the fan T.T

Help please T.T

**Comment by Aissasa** [comment]
Hey, I started on my file from scratch, and made it work; I still dont know why the initial one wasnt working, maybe some privilege issue or something.
[Here](https://pastebin.com/g8EB0Utu)'s the updated version (still working on it, but it's functional)

Cheers yall ^^

**Comment by owallb** [comment]
Hey, I had a similar issue and figured I may as well share what I found for future people:

```
oscar@home docker-qbittorrent $ docker compose up -d  
[+] Running 3/5
 ✔ Network docker-qbittorrent_default  Created
 ⠧ Container jackett                   Starting
 ✘ Container gluetun                   Error
 ⠧ Container flaresolverr              Starting
 ✔ Container qbittorrent               Created
Error response from daemon: failed to create task for container: failed to create shim task: OCI runtime create failed: runc create failed: unable to create new parent process: na
mespace path: lstat /proc/3648302/ns/net: no such file or directory
```

I then checked the logs for the gluetun container which gave a good hint:

```
oscar@home docker-qbittorrent $ docker compose logs gluetun                                                                                                                      1
[...]
gluetun  | 2025-03-26T05:32:32Z ERROR reading VPN settings: VPN provider: server selection: environment variable WIREGUARD_ENDPOINT_IP: ParseAddr("vpn60.prd.kista.ovpn.com"): unexpected character (at "vpn60.prd.kista.ovpn.com")
gluetun  | 2025-03-26T05:32:32Z INFO Shutdown successful
```

So it didn't like that I used a FQDN for my endpoint. After changing it to the actual IPv4 address it worked.
